package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "onboard-javascript",
+  "version": "1.0.0",
+  "description": "Onboarding",
+  "main": "index.js",
+  "dependencies": {
+    "jquery": "^3.4.1"
```

Comment:
Pls include typescript as an npm package, rather than with apt. The idea is that someone should be able to clone this repo, run _npm install_ and have everything they need, rather than running into the error when they try to run tsc.
```
npm install --save-dev typescript 
``
The _--save-dev_ script saves an entry for the version and for the name of the package.